src/lib/ext-listen.js

```diff
@@ -31,3 +31,19 @@ export function openActivityLogPage() {
     url: getActivityLogPageURL(),
   });
 }
+
+export function dateTimeFormat(timestamp, options) {
```

Comment:
Why did you put this function in `ext-listen.js` instead of a new file called, say, `formatters.js`?